.gitignore

```diff
@@ -1,4 +1,9 @@
 # build directories
 build/
 build_lnx/
-build_win/
\ No newline at end of file
+build_win/
+/__intel/*
+/binaries_to_zip/*
+/lib*
+/test_conformance/intel/*
+/test_conformance/gl/GL/*
```

Comment:
I'm not sure all of these really belong upstream.

Reply:
Yes, my mistake. Thanks.